data/init.sql

```diff
@@ -137,34 +137,34 @@ CREATE TABLE IF NOT EXISTS courses (
 	updated_at TIMESTAMP
 ); 
 
-CREATE TABLE IF NOT EXISTS course_offerings(
+CREATE TABLE IF NOT EXISTS lectures (
 	id INT(11) UNSIGNED AUTO_INCREMENT PRIMARY KEY,
 	course_id INT(11) UNSIGNED,
 	FOREIGN KEY(course_id) REFERENCES courses(id) ON DELETE CASCADE,
-	course_offering_code VARCHAR(60),
-	course_offering_name VARCHAR(60),
+	lecture_code VARCHAR(60),
+	lecture_name VARCHAR(60),
```

Comment:
not sure we need a lecture code and a lecture name, I think just lecture code would do

Reply:
yeah i agree, the name of the course is already in the courses table, so i dont think we need a lecture name

Reply:
I think we can do without it, agreed. Each having a name and code gets cumbersome. Will put up a version with them removed. Might require additional refactoring of menus to make them show the course name but otherwise shouldn't effect very much.